src/styles/youtubeMockup.scss

```diff
@@ -0,0 +1,295 @@
+//YouTube Mockup Styles
```

Comment:
Careful mixing tailwind and full blow style files like this. I see areas in here where is makes sense to put it in a scss file opposed to using tailwind, but I also see styles that would be easier to use tailwind for.